core/src/botany_core/runner.py

```diff
@@ -44,7 +44,7 @@ def rerun_game(game, move_list):
     return boards
 
 
-def run_game(game, fn1, fn2, opcode_limit=None, display_board=False):
+def run_game(game, fn1, fn2, opcode_limit=None, display_board=False,move_list=[]):
```

Comment:
Using a list as a default argument is almost never what you want in Python.  [See this article](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) for more.

Reply:
@inglesp Thanks for the link!